src/components/Signup/Signup.js

```diff
@@ -152,18 +152,20 @@ const Signup = function({ intl, dependencies: { dopplerLegacyClient, originResol
             <FieldGroup>
               <InputFieldItem
                 autoFocus
+                type="text"
                 className="field-item--50"
                 fieldName={fieldNames.firstname}
                 label={_('signup.label_firstname')}
-                type="text"
                 required
+                isValidateName
```

Comment:
What do you think about simply `validateName`?

Reply:
Or perhaps `isValidName`?

Reply:
But, it does not represent a state, for that reason, I think that _is_ prefix is wrong.

It is a configuration of the input component.

Reply:
😴 💤 💤

Reply:
Because is the same name of the function to validate

Reply:
needValdiateName ?

Reply:
_withNameValidation_?

Reply:
_isName_?

Reply:
withNameValidation like mroe